src/main/java/jesseg/ibmi/opensource/nginx/NginxConfNode.java

```diff
@@ -39,9 +41,18 @@ public static NginxConfNode open(final File _file) throws IOException {
                     curStr = "";
                 } else if (';' == c) {
                     curStr = curStr.trim();
-                    final String property = curStr.replaceAll("\\s.*", "");
-                    final String value = curStr.replaceFirst("^[^\\s]*\\s+", "");
-                    currentNode.addProperty(property, value);
+                    Pattern pattern = Pattern.compile("^([^\\s]+)\\s*($|.*$)"); //"^(\\S*)\\s*($|.*$)"
+                    Matcher matcher = pattern.matcher(curStr);
+                    String property = null;
+                    String value = null;
+                    while (matcher.find()) {
```

Comment:
This should be an `if` instead of a `while`. The logic doesn't make any sense if we're iterating here.